service/src/main/java/org/apache/celeborn/server/common/service/config/FsConfigServiceImpl.java

```diff
@@ -88,10 +85,12 @@ private synchronized void refresh() {
       }
     } catch (Exception e) {
       LOG.warn("Refresh dynamic config error: {}", e.getMessage(), e);
+      return;
```

Comment:
this silently changes the logic, but looks fine

Reply:
@RexXiong, @pan3793, I have updated the `What changes were proposed in this pull request?` to mention this change.

Reply:
I rethought that this change might cause systemConf to be null